File: CSharp_ChildrenCompetitionSockets/CSharp_ChildrenCompetitionSockets/model/TestType.cs
using System;

namespace CSharp_ChildrenCompetitionGUI.model
{
    [Serializable]
    public class TestType : Entity<int>
    {
        //DRAWING, TREASURE_HUNT, POETRY
        public string type { get; }

        public TestType(string type)
        {
            this.type = type;
        }
    }
}

File: Java_ChildrenCompetitionSockets/CompetitionModel/src/main/java/competition/model/TestType.java
package competition.model;

public class TestType extends Entity<Integer>{
    //DRAWING, TREASURE_HUNT, POETRY
    private String type;

    public TestType() {

    }
    public TestType(String type) {
        this.type = type;
    }

    public String getType() {
        return type;
    }
}
